experimental/unicodeset_parse/src/parse.rs

```diff
@@ -1344,6 +1364,50 @@ where
         self.single_set.add_set(&set.to_code_point_inversion_list());
         Ok(())
     }
+
+    fn try_load_grapheme_cluster_break_set(&mut self, name: &str) -> Result<()> {
```

Comment:
nit: this code is pretty repetitive, but I guess the existing code also is

Reply:
If I should use macro instead of the following, I will replace with it. Should I use macro like it?
```rust
macro_rules! impl_try_load_property_set {
    ($func:ident, $property: ident, $loadfn: ident) => {
        fn $func(&mut self, name: &str) -> Result<()> {
            let name_map = $property::get_name_to_enum_mapper(self.property_provider)
                .map_err(|_| PEK::Internal)?;
            let value = name_map
                .as_borrowed()
                .get_loose(name)
                .ok_or(PEK::UnknownProperty)?;
            // TODO(#3550): This could be cached; does not depend on name.
            let property_map = $loadfn(self.property_provider).map_err(|_| PEK::Internal)?;
            let set = property_map.as_borrowed().get_set_for_value(value);
            self.single_set.add_set(&set.to_code_point_inversion_list());
            Ok(())
        }
    };
}
```

Reply:
You can't define the macro here because it's inside an impl, so I think it's easier to just leave it as it is.

Reply:
OK.